doc/apiary/v2/fiware-ngsiv2-2.0-2018_09_15.apib

```diff
@@ -35,10 +35,64 @@ Sergio García Gómez (Telefónica I+D), Martin Bauer (NEC).
   
 ## Status
 
-This is a work in progress and is changing on a daily basis.
-Please send your comments to fiware-ngsi@lists.fiware.org.
-You can trace the discussions checking the archives of the mailing list:
-https://lists.fiware.org/private/fiware-ngsi/ (list subscription required).
+This specification is the final and stable version of NGSIv2 API specification (v2.0).
+
+# Changelog
+
+Changes since RC-2018.07:
+
+* None
+
+Changes since RC-2008.04:
+
+* Section "System/builtin Attributes" renamed to "Builtin Attributes"
+* Section "System/builtin Metadata" renamed to "Builtin Metadata"
+* Added new builtin attribute: `dateExpires`
+* Improved description in "Error Response" section
+* Added new option for `POST /v2/entities` operation: `upsert`
+* Clarifying about forwarding compliance in "Group Registrations"
```

Comment:
Clarification about ...

Reply:
Fixed in db5b589368f93c94cdc5d6b4cb24ee3a19398416